source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -285,6 +297,37 @@ class StreamHandleWrapper : public Filters::Common::Lua::BaseLuaObject<StreamHan
   State state_{State::Running};
   std::function<void()> yield_callback_;
   Http::AsyncClient::Request* http_request_{};
+  FireAndForgetHttpWriter* fireAndForgetHttpWriter_;
+};
+
+/**
+ * HTTP client that performs non-blocking HTTP calls and ignores the response.
+ */
+class FireAndForgetHttpWriter : public Filters::Common::Lua::BaseLuaObject<FireAndForgetHttpWriter>,
+                                public Http::AsyncClient::Callbacks {
+public:
+  FireAndForgetHttpWriter(Filter& filter);
```

Comment:
use `explicit` for single arg ctors: https://google.github.io/styleguide/cppguide.html#Implicit_Conversions